### ISSUE TITLE
MachAr

### ISSUE THREAD

**Issue by gdmcbain** [gdmcbain/DySys#5]
Downstream report from gdmcbain/millihydraulics#1.

**Comment by gdmcbain** [comment]
And indeed it's the same here:

```PowerShell
python -m pytest
```

results in

```text

=============================================== short test summary info ===============================================
ERROR tests/fixed_point/test_fixed_point.py - AttributeError: module 'numpy' has no attribute 'MachAr'
ERROR tests/ladder/test_ladder.py - AttributeError: module 'numpy' has no attribute 'MachAr'
ERROR tests/lumped_line/test_lumped_line.py - AttributeError: module 'numpy' has no attribute 'MachAr'
ERROR tests/scalar/test_scalar.py - AttributeError: module 'numpy' has no attribute 'MachAr'
ERROR tests/signal_flow_path/test_signal_flow_path_sys.py - AttributeError: module 'numpy' has no attribute 'MachAr'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 5 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================== 5 errors in 1.72s ==================================================
```

**Comment by gdmcbain** [comment]
https://numpy.org/doc/stable/release/1.22.0-notes.html#the-np-machar-class-has-been-deprecated

**Comment by gdmcbain** [comment]
Closed by #9
